packages/webapp/src/webview/ui/components/Header/Filters/FiltersRibbon.tsx

```diff
@@ -32,17 +32,44 @@ export function FiltersRibbon() {
         });
     };
 
+    const usePrevious = (value: any) => {
```

Comment:
Why do we need to add/change productive code when changing the test framework?

Reply:
I reverted the changes in this file, functionality can be tested in a different way. Thanks!